distribution/docs/src/main/jdocs/content/_running/starting-intro-contents.adoc

```diff
@@ -170,22 +170,20 @@ LimitNOFILE=6815744
 #Add the following:
 wrapper.java.additional.11=-D${ddf-branding-lowercase}.home=%KARAF_HOME%
 wrapper.java.additional.12=-Dderby.storage.fileSyncTransactionLog=true
-wrapper.java.additional.13=-server
```

Comment:
[KARAF-5094](https://issues.apache.org/jira/browse/KARAF-5094) Remove -server option in Karaf scripts